setup_lacework_agent.sh

```diff
@@ -66,7 +66,7 @@ render_agent_config() {
 
   # Additional Config Json
   if [ "$ADDITIONAL_CONFIG" != "" ]; then
-    _additional_config_json='$ADDITIONAL_CONFIG,'
+    _additional_config_json='"'$ADDITIONAL_CONFIG'",'
```

Comment:
Do we have some kind of functional test somewhere?

The impact of this module silently not working can be bad 😅

Reply:
Unfortunately we do not have an automated test to verify the config file  is correct :(
I have tested this manually. 

But will reach out to the agent team to see if we can add some improved testing.